.github/workflows/runtime-version-scanner.yml

```
@@ -0,0 +1,87 @@
+name: Runtime Version Scanner
+
+# Re-usable workflows use the `workflow_call` trigger
+# https://docs.github.com/en/actions/sharing-automations/reusing-workflows#creating-a-reusable-workflow
+on:
+  workflow_call:
+
+# Setup the environment with the extension name for easy re-use
+# Also set the GH_TOKEN for the release-extension action to be able to use gh
+env:
+  EXTENSION_NAME: runtime-version-scanner
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  extension:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./extensions/${{ env.EXTENSION_NAME }}
+
+    steps:
+      # Checkout the repository so the rest of the actions can run with no issue
+      - uses: actions/checkout@v4
+
+      # We want to fail quickly if the linting fails, do that first
+      - uses: ./.github/actions/lint-extension
+        with:
+          extension-name: ${{ env.EXTENSION_NAME }}
+
+      # ---
+      # Run R tests
+      # ---
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: '4.3.3'
+
+      - uses: r-lib/actions/setup-renv@v2
+        working-directory: extensions/${{ env.EXTENSION_NAME }}
+
+      - run: Rscript -e 'testthat::test_dir("tests/testthat")'
+        working-directory: extensions/${{ env.EXTENSION_NAME }}
```

Ah I think I found it

This needs to be the below I believe

```suggestion
      - uses: r-lib/actions/setup-renv@v2
        with:
          working-directory: extensions/${{ env.EXTENSION_NAME }}

      - run: Rscript -e 'testthat::test_dir("tests/testthat")'
        with:
          working-directory: extensions/${{ env.EXTENSION_NAME }}
```

---

Thank you!!